builds/misc/images.yaml

```diff
@@ -354,6 +368,13 @@ jobs:
           imageName: azureiotedge-module-restarter
           project: ModuleRestarter
 
+     # Metrics Collector
+      - template: templates/image-windows.yaml
+        parameters:
+          name: Metrics Collector
+          imageName: azureiotedge-metrics-collector
+          project: MetricsCollector
```

Comment:
Are we not publishing the docker binary for Metric Collector?

Reply:
I feel we should discuss publishing via releases after we have verified the metrics collector is stable (through building the dashboard).